pkg/bot/slack.go

```diff
@@ -47,11 +47,8 @@ type Slack struct {
 
 	Client           *slack.Client
```

Comment:
Hmm 🤔 I tried to test it and:

```
      channels:
        'default':
          # -- Slack channel name without '#' prefix where you have added BotKube and want to receive notifications in.
          name: 'SLACK_CHANNEL'
          bindings:
            executors:
              - kubectl-read-only
              - kubectl-read-only2
            sources:
              - k8s-events`
```

```
executors:
  'kubectl-read-only2':
    ## Kubectl executor configuration.
    kubectl:
      namespaces:
        # -- List of allowed Kubernetes Namespaces for command execution. The "all" keyword, allows all the Namespaces.
        include:
          - all
        # -- List of ignored Kubernetes Namespace. It can be used only when `include: ["all"]`.
        # It can contain a wildcard (*) that expands to zero or more arbitrary characters
        # Example: `ignore: ["x", "y", "secret-ns-*"]`
        ignore: []
      # -- If true, enables `kubectl` commands execution.
      enabled: true
      ## List of allowed `kubectl` commands.
      commands:
        # -- Configures which `kubectl` methods are allowed.
        verbs: ["get", "logs", "top", "auth"]
        # -- Configures which K8s resource are allowed.
        resources: ["deployments", "pods"]
      # -- Configures the default Namespace for executing BotKube `kubectl` commands. If not set, uses the 'default'.
      defaultNamespace: default
      # -- If true, enables commands execution from configured channel only.
      restrictAccess: true
  'kubectl-read-only':
    ## Kubectl executor configuration.
    kubectl:
      namespaces:
        # -- List of allowed Kubernetes Namespaces for command execution. The "all" keyword, allows all the Namespaces.
        include:
          - all
        # -- List of ignored Kubernetes Namespace. It can be used only when `include: ["all"]`.
        # It can contain a wildcard (*) that expands to zero or more arbitrary characters
        # Example: `ignore: ["x", "y", "secret-ns-*"]`
        ignore: []
      # -- If true, enables `kubectl` commands execution.
      enabled: true
      ## List of allowed `kubectl` commands.
      commands:
        # -- Configures which `kubectl` methods are allowed.
        verbs: ["api-resources", "api-versions", "cluster-info", "describe", "diff", "explain"]
        # -- Configures which K8s resource are allowed.
        resources: [ "namespaces", "daemonsets", "statefulsets", "storageclasses", "nodes", "configmaps"]
      # -- Configures the default Namespace for executing BotKube `kubectl` commands. If not set, uses the 'default'.
      defaultNamespace: default
      # -- If true, enables commands execution from configured channel only.
      restrictAccess: true
```

<img width="753" alt="Screenshot 2022-08-04 at 13 22 10" src="https://user-images.githubusercontent.com/7155799/182835179-13375534-554d-4975-81ae-2e6b38318402.png">

<img width="755" alt="Screenshot 2022-08-04 at 13 22 18" src="https://user-images.githubusercontent.com/7155799/182835198-ea7a80e0-d5e5-4457-b0f4-56999d9e371d.png">


Looks like the `kubectl-read-only2` binding doesn't work, even if I have that in `commands list`:
<img width="444" alt="Screenshot 2022-08-04 at 13 22 59" src="https://user-images.githubusercontent.com/7155799/182835299-2bba76a7-f372-4a12-a528-890b96832d23.png">

Do you know what could be the problem?

Reply:
Alright, that was issue on my side, thanks for the sync 😄